cmd/soroban-rpc/internal/ledgerbucketwindow/ledgerbucketwindow.go

```diff
@@ -66,11 +66,11 @@ func (w *LedgerBucketWindow[T]) Len() uint32 {
 }
 
 // Get obtains a bucket from the window
-func (w *LedgerBucketWindow[T]) Get(i uint32) *LedgerBucket[T] {
+func (w *LedgerBucketWindow[T]) Get(i uint32) (*LedgerBucket[T], error) {
 	length := w.Len()
 	if i >= length {
-		panic(fmt.Errorf("index out of range [%d] with length %d", i, length))
+		return nil, fmt.Errorf("index out of range [%d] with length %d", i, length)
```

Comment:
I think panics like these are justified (just like `s[i]` panics if `i` is out of bounds).

Reply:
Yeah makes sense